### PR TITLE
Optimize read ddl

### PR DESCRIPTION
**Pull request by longjiquan**
Signed-off-by: longjiquan <jiquan.long@zilliz.com>
issue: #19097

### CODE REVIEW

File: internal/rootcoord/root_coord.go [hunk 1]
```diff
@@ -819,52 +821,56 @@ func (c *Core) HasCollection(ctx context.Context, in *milvuspb.HasCollectionRequ
...
-	t := &hasCollectionTask{
-		baseTaskV2: baseTaskV2{
-			ctx:  ctx,
-			core: c,
-			done: make(chan error, 1),
-		},
-		Req: in,
-		Rsp: &milvuspb.BoolResponse{},
-	}
+	log.Info("received request to has collection")
```
Comment (czs007) [inline internal/rootcoord/root_coord.go:827]: Can we use log.Ctx ?

Comment (longjiquan) [reply internal/rootcoord/root_coord.go:827]: OK, I'll fix this.

**Pull request merged**